app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -130,14 +249,41 @@ def params_for_attach
   def self.raw_create_volume(ext_management_system, options)
     volume = nil
     volume_params = nil
+    affinity_volume = nil
+    anti_affinity_volumes = nil
+    affinity_pvm_instance = nil
+    anti_affinity_pvm_instances = nil
+
+    affinity_policy = options['affinity_policy'] == 'Off' ? nil : options['affinity_policy']
+
+    case affinity_policy
```

Comment:
Codeclimate doesn't like the added complexity of this `case` switch I added. I'm open to ideas to improve it, if anyone had any :shrug: